src/planscape/forsys/parse_forsys_output.py

```diff
@@ -0,0 +1,160 @@
+import rpy2
+
+import numpy as np
+
+from typing import TypedDict
+
+
+class RankedProject(TypedDict):
+    # Project ID.
+    id: int
+    # Contribution of each priority to the total score
+    # The contribution is weighted according to a scenario's priority weights.
+    weighted_priority_scores: dict
+    # The total score, summed across weighted priority scores.
+    total_score: float
+    # Project rank.
+    rank: int
+
+
+class Scenario(TypedDict):
+    # Priority weights for the scenario.
+    priority_weights: dict
+    # A list of the projects, ranked.
+    ranked_projects: list[RankedProject]
+    # Given ranked projects, a cumulative sum of project area.
+    cumulative_ranked_project_area: list[float]
+    # Given ranked projects, a cumulative sum of project cost.
+    cumulative_ranked_project_cost: list[float]
+
+
+# Transforms the output of a Forsys scenario set run into a more easily-interpreted version.
+class ForsysScenarioSetOutput():
+    scenarios: dict
+
+    # Converts a ForsysScenarioSetOutput to a dictionary.
+    # This facilitates JSON conversion.
+    def to_dictionary(self) -> dict:
```

Comment:
Does simply returning the "scenarios" dictionary work?

Reply:
definitely - removed this.